src/components/ColorPicker/ColorResult.vue

```diff
@@ -131,4 +131,10 @@ export default {
     }
   }
 }
+
+@media only screen and (max-width: 640px) {
```

Comment:
Hi, can you make max-width to `(max-width: 768px)` like i did.

Reply:
I've made changes, thanks for correcting me. i think it's better now

Reply:
Thank you